pkg/api/heartbeat.go

```diff
@@ -135,7 +135,7 @@ func parseHeartbeatResponse(data []json.RawMessage) (heartbeat.Result, error) {
 		return heartbeat.Result{}, fmt.Errorf("failed to parse json status: %s", err)
 	}
 
-	if result.Status >= http.StatusBadRequest {
+	if result.Status < 200 || result.Status > 299 {
```

Comment:
Can we use the builtin constant? @alanhamlett

Reply:
I wanted to, but there's no constant for `299` and the constant for `300` (`http.StatusMultipleChoices`) doesn't sound like an error so it's more confusing than just using `299`.

Reply:
We can use at least for Status Ok?

Reply:
Done!